packages/block-editor/src/components/block-alignment-control/ui.js

```diff
@@ -53,42 +70,106 @@ function BlockAlignmentUI( {
 	isCollapsed = true,
 } ) {
 	const enabledControls = useAvailableAlignments( controls );
-	if ( enabledControls.length === 0 ) {
+	const hasEnabledControls = !! enabledControls.length;
+
+	if ( ! hasEnabledControls ) {
 		return null;
 	}
+	const wideAlignmentsSupport = enabledControls.some( ( { name } ) =>
+		WIDE_CONTROLS.includes( name )
+	);
+	// Always add the `none` option if not exists.
+	if ( ! enabledControls.some( ( { name } ) => name === 'none' ) ) {
```

Comment:
Do you think it's possible to remove this and move the responsibility to the layout to always include "none". In other words, since we can't really remove "none" from the return value of `getAlignments` of layouts, we should just consider it a regular alignment with `undefined` as value.

Reply:
Addressed in https://github.com/WordPress/gutenberg/pull/34710/commits/292ead1e3d9ce3646056c34a42ea1f937ff72c3d